recipes/pcl/all/conanfile.py

```diff
@@ -169,7 +171,13 @@ def _configure_cmake(self):
     def build(self):
         self._patch_sources()
         cmake = self._configure_cmake()
-        cmake.build()
+        if self.settings.compiler == "clang":
+            # Workaround for OOM when building with clang. 
+            # See https://github.com/conan-io/conan-center-index/pull/1891 for discussion
+            compile_jobs = os.getenv("CONAN_CPU_COUNT", int(math.ceil(tools.cpu_count() / 2)))
+            self.run(f"cmake --build {cmake.build_folder} {cmake.build_config} -- -j{compile_jobs}")
```

Comment:
maybe something like this:

```python
compile_jobs = os.getenv("CONAN_CPU_COUNT", int(math.ceil(tools.cpu_count() / 2)))
with tools.environment_append({"CONAN_CPU_COUNT": compile_jobs}):
    cmake.build()

Reply:
Yes that's better...but didn't work anyway :(